tools/buildsys/src/main.rs

```diff
@@ -98,7 +102,8 @@ USAGE:
 
 SUBCOMMANDS:
     build-package           Build RPMs from a spec file and sources.
-    build-variant           Build filesystem and disk images from RPMs."
+    build-variant           Build filesystem and disk images from RPMs.
+    variant-paths           Get a list of all source paths that contribute to a variant."
```

Comment:
What is interesting about this command is that it does not need to be invoked from a `build.rs` file. In my mind the purpose of `buildsys` is to be invoked by `cargo` from ` build.rs` process. This command could be `twoliter info variant-paths` or something like that instead of being in buildsys. It would require moving common code somewhere or making a `buildsys` library.

Reply:
Oh wait, maybe it does. I need to look more closely.

Reply:
The idea was this could be invoked directly, but perhaps a `twoliter info` subcommand could be a better fit.

The motivation for `buildsys` was the existing logic in there for understanding `Cargo.toml` files and some of their relationships. The path identification is related but somewhat different than the existing code, so I don't think we ended up with as much overlap and mutually beneficial gains from being in `buildsys` though. It's still logically similar and easier to group that way, but I don't think it necessarily needs to be if there is a better place for this logic.

Reply:
The manifest parser is already a crate, and the spec parser and project walker could be crates too, if we want to shift this into `twoliter` or some other tool.

I would start by making it a standalone tool though, or keeping it in `buildsys`, and invoking via `Makefile.toml`. I want to make sure we have a consistent vision and agreement on what subcommands `twoliter` grows, since that CLI is our future API.